createdisk.sh

```diff
@@ -53,9 +53,6 @@ VM_IP=$(sudo virsh domifaddr ${VM_NAME} | tail -2 | head -1 | awk '{print $4}' |
 wait_for_ssh ${VM_NAME} ${VM_IP}
 
```

Comment:
>  we need to cached a specific image after prune (baremetal-runtimecfg) to run node-ip command.

What is this baremetal-runtimecfg/node-ip command? What do they achieve, when do they run?